rll/enable-managed-login.sh

```diff
@@ -195,12 +201,15 @@ enable)
     sudo bash -c "umask 0337 && printf '# Members of the rightscale_sudo group may gain root privileges\n%%rightscale_sudo ALL=(ALL) SETENV:NOPASSWD:ALL\n' > /etc/sudoers.d/90-rightscale-sudo-users"
   fi
 
+  # Ask user input what should be the homedir permission. If no input is given, set default value (0026) which is eq to 7751
+  : "${UMASK:=0026}"
```

Comment:
Same as above.

Reply:
PR updated, default umask back to `0022`